docs/settingUp.md

```diff
@@ -24,7 +24,7 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
    ```sh
    git remote add upstream https://github.com/TEAMMATES/teammates.git
    ```
-  
```

Comment:
This is an auto-generated change, I don't know why it changes. And it seems that the content doesn't change.